runtimeinfo/noflo.yaml

```diff
@@ -70,10 +70,10 @@ componenttemplates:
         output.done()
 
   es2015: |
-    import noflo from 'noflo';
+    const noflo = require('noflo');
 
-    export var getComponent = function() {
-      var c = new noflo.Component();
+    exports.getComponent = () => {
+      const c = new noflo.Component();
```

Comment:
I prefer using `const` for things which should be immutable. Consider using `let` instead?

Reply:
@jonnor this is required by the AirBnB ESLint rules we use in most projects. We're not changing the value of `c`, but we're definitely interacting with its contents

Reply:
Silly, but OK.